moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -55,10 +55,13 @@
 
 
 class PlanningSceneInterface(object):
-    """ Simple interface to making updates to a planning scene """
+    """ 
+    Python interface for a planning scene.
```

Comment:
Does this provide an interface to modifying **a** `PlanningScene` or to modifying **the** `PlanningScene` curated by the `PlanningSceneMonitor`? (Not sure if the first one even exists in Python)

Reply:
True, it's the latter. A Python interface for the `PlanningSceneInterface`, not the `PlanningScene`.

There are no Python bindings for the `PlanningScene`, although there is one for the `RobotState`. There has been discussion in #900 and I should probably raise the issue again.